kustomize/base/nginx/templates/routes/openai.conf.template

```diff
@@ -1,10 +1,11 @@
+
+
 location /openai {
 
    location ~* ^\/openai\/v1\/((engines\/.+\/)?(?:chat\/completions|completions|edits|moderations|answers|embeddings))$ {
       rewrite ^\/openai(\/.*)$ $1 break;
-      proxy_pass https://api.openai.com;
+      proxy_pass ${NGINX_UPSTREAM_NIM_LLM};
```

Comment:
if this is for the open ai interface, you should use the openai environment variable as suggested in my previous comment.

Reply:
addressed as agreed.